packages/config/src/layer3s/syndicateframe.ts

```diff
@@ -0,0 +1,31 @@
+import { ProjectId } from '@l2beat/shared-pure'
+
+import { underReviewL3 } from '../layer2s/templates/underReview'
+import { Layer3 } from './types'
+
+export const syndicateframe: Layer3 = underReviewL3({
+  id: 'syndicateframe',
+  hostChain: ProjectId('base'),
+  display: {
+    name: 'Syndicate Frame Chain',
+    slug: 'syndicateframe',
+    category: 'Optimium',
+    description:
+      'Syndicate Frame Chain is an OP Stack L3 on Base with Celestia underneath unlocking high throughput and extremely low gas costs for Frame developers.',
+    purposes: ['Social', 'NFT'],
+    provider: 'Arbitrum Orbit',
```

Comment:
Shouldn't the provider be OP Stack? According to the description

Reply:
yes, sorry!